edi_product_oca/views/product_views.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="product_template_form_view" model="ir.ui.view">
+        <field name="name">product.template.form.view - edi_product_oca</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view" />
+        <field name="arch" type="xml">
+
+            <page name="inventory" position="after">
+                <page name="edi" string="EDI">
+                    <group
+                        name="edi_product_group"
+                        string="Electronic Data Interchange"
```

Comment:
```suggestion
```
this is a whole page for EDI, no need to be so verbose IMO

Reply:
I personally preferred how it looked this way:

![image](https://github.com/OCA/edi-framework/assets/123451545/a3f92808-481c-4e2b-bb4b-246276b7c67b)

Better than how it looks for example in _edi_sale_oca_ without group name:

![image](https://github.com/OCA/edi-framework/assets/123451545/76e424c7-26a4-404e-828e-29b2294f8575)

But nvm, already changed it! 👍🏼